lint-tasks.yml

```
@@ -103,7 +103,9 @@ tasks:
     cmds:
       - task: ":utils:clang-tidy"
         vars:
-          FLAGS: "--config-file={{.ROOT_DIR}}/.clang-tidy -p {{.CLP_FFI_PY_COMPILE_COMMANDS_DB}}"
+          FLAGS: >
+            --config-file={{.CLP_FFI_PY_CPP_SRC_DIR}}/.clang-tidy
+            -p {{.CLP_FFI_PY_COMPILE_COMMANDS_DB}}
```

```suggestion
          FLAGS: >-
            --config-file "{{.CLP_FFI_PY_CPP_SRC_DIR}}/.clang-tidy"
            -p "{{.CLP_FFI_PY_COMPILE_COMMANDS_DB}}"
```

* As our [guidelines](https://docs.yscope.com/dev-guide/contrib-guides-taskfiles.html#paths-in-commands) mention, paths in commands should be quoted.
* Instead of `=` for one arg and space-separation for the other, let's use one consistent style.

---

kk, sorry I didn't realize there's a guideline for task files. Will read it soon